### DISCUSSION TITLE
Where is traccar.xml ??

### DISCUSSION THREAD

**Discussion by nwb-usr** [hassio-addons/addon-traccar#290] [Q&A] [answered]
Hi, I'm traying to edit traccar.xml but i cannot find it anywhere. Some help?

(running Home Assistant Core 2023.11.2 on Home Assistant OS 11.1 on Home Assistant Blue HW aka ODROID N2+)

**Comment by lordrak007** [comment]
Hi,
check addon traccar folder...
HA created config folder for every addon with config. So search folder first, then traccar config file.

**Comment by nwb-usr** [comment]
Thanks for the answer.  That is where I get lost.
I searched for it with no success.  Searched in /config, in /addons /media /share etc..

I tryed with `find / -type f -name traccar.xml` command but nothing.
Should I create it somewhere??

**Reply by nwb-usr** [reply, marked as answer]
ok, I just found that traccar.sh file says 'if ! bashio::fs.file_exists "/config/traccar.xml"; then
    cp /etc/traccar/traccar.xml /config/traccar.xml'

So created traccar.xml in /config and done.  It works.

If anyone else needs to modify it but it is missing, just create it in /config (copy it from [HERE](https://github.com/hassio-addons/addon-traccar/blob/main/traccar/rootfs/etc/traccar/traccar.xml) ).

Again was on Home Assistant Core 2023.11.2 on Home Assistant OS 11.1 on ODROID n2+.

Thanks again.
